README.md

```diff
@@ -13,6 +13,12 @@ and away you go with a full-stack blockchain and command line tool for transacti
 
 WARNING: Currently uses plain-text private keys for transactions and is otherwise not production ready.
 
+## Prerequisites
+
+* Go to https://golang.org/doc/install to install Golang. 
+* Once you have Golang installed you will also need to install the Golang package manager `glide`: https://github.com/Masterminds/glide. 
```

Comment:
there is no need to `make get_vendor_deps` if `go get -u .../cmd/basecoin` succeeded.